gcc/testsuite/gcc.dg/tree-ssa/ssa-store-ccp-2.c

```diff
@@ -1,5 +1,6 @@
 /* { dg-do compile } */
 /* { dg-options "-O2 -fdump-tree-optimized" } */
+/* { dg-options "-O2 -fdump-tree-optimized -mno-sdata" { target arc*-*-* } } */
```

Comment:
The triple should only select the ELF toolchain as for uClibc we do not use sdata.